Sources/SmileID/Classes/DocumentVerification/View/DocumentConfirmationView.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+
+struct DocumentConfirmationView: View {
```

Comment:
Can this be generalized with the existing selfie capture confirmation? i.e. a generic image capture confirmation component.

something like: https://github.com/smileidentity/android/blob/main/lib/src/main/java/com/smileidentity/compose/components/ImageCaptureConfirmationDialog.kt

Reply:
Bump